### ISSUE TITLE
Possible item changes

### ISSUE THREAD

**Issue by NeunEinser** [NeunEinser/bingo#235]
- Armadillo scutes / wolf armor
- potato / carrot / golden carrot
- remove tnt items from sand category
- Maybe add a type of colored glass?
- Maybe include moss carpet in mangrove category?
- carpet and banner as a wool item?
- Loom as alternative to bows?
- Change fish to cooked?
- jungle sapling and split sapling category
- remove some diamond / gold tools to improve blind runs

**Comment by BastLast** [comment]
What would be better is allow the user to customize the items

**Comment by NeunEinser** [comment]
@BastLast As I explained there, it is impossible from a technical perspective to add an in-game way for this.

**Comment by BastLast** [comment]
> @BastLast As I explained there, it is impossible from a technical perspective to add an in-game way for this.

yup i saw that in the other story after posting my comment 😅, loved the idea of having different presets tho.

congratulation on the map btw been playing with a few friends for a few hours and i've been impressed by the quality of the work :) Hope a 1.21 version will be done !
